src/app/_helpers/factories/building.factory.ts

```diff
@@ -0,0 +1,13 @@
+import {ComponentFactory, ComponentFactoryResolver, FactoryProvider, OnInit} from '@angular/core';
+import { Building} from '../../_models/building.model';
+import {FactoryGenerator} from '@angular/compiler-cli/src/ngtsc/shims';
+
+export class BuildingFactor {
```

Comment:
@HonzaDudek typo

Reply:
Created building factory and corrected typo